opentelemetry-sdk/README.rst

```diff
@@ -13,6 +13,23 @@ Installation
 
     pip install opentelemetry-sdk
 
+Running Performance Tests
```

Comment:
Curious does this belong in the sdk README? I feel like it would be useful in the top level.

Also, unrelated but should be in the same place as a "HOW TO" for writing benchmark tests right?

Reply:
Yes I agree I think it would be better at the top level! Since running the tests is different from developing tests, I can move it there.

> Also, unrelated but should be in the same place as a "HOW TO" for writing benchmark tests right?

Hm, I guess we could write a "How to write Resource Utilization Tests" section [like we did for benchmarks here](https://github.com/open-telemetry/opentelemetry-python/blob/master/CONTRIBUTING.md#benchmarks). However, since it doesn't require a specific format like the benchmarks did (because you can **just profile any Python program**, I just provided examples here) I can't immediately think of any meaningful guidelines to provide.